inc/namespace.php

```diff
@@ -14,6 +14,7 @@
  */
 function bootstrap() {
 	About\bootstrap();
+	Core\Consent\bootstrap();
```

Comment:
We don't have a specific `Core` namespace, just the root namespace `Altis`
```suggestion
	Consent\bootstrap();
```

Reply:
See above https://github.com/humanmade/altis-core/pull/92#discussion_r488944265